payjoin/src/core/receive/v2/mod.rs

```diff
@@ -985,6 +1099,41 @@ pub mod test {
         Ok(())
     }
 
+    #[test]
+    fn test_extract_err_req_expiry() -> Result<(), BoxError> {
+        let now = SystemTime::now();
+        let noop_persister = NoopSessionPersister::default();
+        let context = SessionContext { expiry: now, ..SHARED_CONTEXT.clone() };
+        let receiver = Receiver {
+            state: UncheckedProposal {
+                v1: crate::receive::v1::test::unchecked_proposal_from_test_vector(),
+                context: context.clone(),
+            },
+        };
```

Comment:
Nit: can we use the helper method defined above? (`unchecked_proposal_v2_from_test_vector`)